src/NuGet.Core/NuGet.Packaging/PackageExtraction/PackageExtractionContext.cs

```diff
@@ -18,5 +20,10 @@ public class PackageExtractionContext
         public PackageSaveMode PackageSaveMode { get; set; } = PackageSaveMode.Defaultv2;
 
         public XmlDocFileSaveMode XmlDocFileSaveMode { get; set; } = PackageExtractionBehavior.XmlDocFileSaveMode;
+
+        public PackageExtractionContext(ILogger logger)
+        {
+            Logger = logger;
```

Comment:
null is valid (tests pass in null). And there's no use of logger here, so no need to check for null.

---
In reply to: [59908891](https://github.com/NuGet/NuGet.Client/pull/504#discussion_r59908891) [](ancestors = 59908891)